autonomy/cli/helpers/deployment.py

```diff
@@ -173,33 +172,6 @@ def build_deployment(  # pylint: disable=too-many-arguments, too-many-locals
     click.echo(report)
 
 
-def update_multisig_address(service_path: Path, address: str) -> None:
-    """Update the multisig address on the service config."""
```

Comment:
This was a very bad idea to begin with because this update the actual service config file